### PR TITLE
TELCODOCS-1348: RN for exposing disk cleaning option in SiteConfig

### PR DESCRIPTION
**Pull request by amolnar-gh**
Version(s): 4.14

Issue: https://issues.redhat.com/browse/TELCODOCS-1348

Link to docs preview: https://63115--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes#ocp-4-14-ztp-siteconfig-disk-cleaning

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -208,6 +208,11 @@ With this release, the Performance Addon Operator (PAO) must-gather image is no
 
 For further information about gathering debugging information relating to low-latency tuning, see xref:../scalability_and_performance/cnf-low-latency-tuning.adoc#cnf-collecting-low-latency-tuning-debugging-data-for-red-hat-support_cnf-master[Collecting low latency tuning debugging data for Red Hat Support].
 
+[id="ocp-4-14-ztp-siteconfig-disk-cleaning"]
+==== Disk cleaning option through SiteConfig and GitOps ZTP
+
+With this release, you can enable removing the partitioning table before intallation with `automatedCleaningMode` in the `SiteConfig` CR. For more information, see xref:../scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-sites.adoc#ztp-deploying-a-site_ztp-deploying-far-edge-sites[Deploying a managed cluster with SiteConfig and GitOps ZTP]
```
Comment (y-first) [inline release_notes/ocp-4-14-release-notes.adoc:214]: Nit: Installation.

File: release_notes/ocp-4-14-release-notes.adoc [hunk 2]
```diff
@@ -245,6 +245,11 @@ With this update, you can provision IPv6 address spoke clusters from dual-stack
 
 With this release, you can pre-cache your application workload images before upgrading your applications on {sno} clusters with {cgu-operator-full}. For more information, see xref:../scalability_and_performance/ztp_far_edge/ztp-talm-updating-managed-policies.adoc#talm-prechache-user-specified-images-concept_ztp-talm[Pre-caching user-specified images with TALM on single-node OpenShift clusters].
 
+[id="ocp-4-14-ztp-siteconfig-disk-cleaning"]
+==== Disk cleaning option through SiteConfig and GitOps ZTP
+
+With this release, you can enable removing the partitioning table before installation with `automatedCleaningMode` in the `SiteConfig` CR. For more information, see xref:../scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-sites.adoc#ztp-deploying-a-site_ztp-deploying-far-edge-sites[Deploying a managed cluster with SiteConfig and GitOps ZTP]
```
Comment (mburke5678) [inline release_notes/ocp-4-14-release-notes.adoc:251]: > enable removing

This sounds odd to me. 
remove the partitioning table
allow the removal of the partitioning table

**Comment by mburke5678** [comment]
@amolnar-rh A couple of nits. Otherwise LGTM

File: release_notes/ocp-4-14-release-notes.adoc [hunk 3]
```diff
@@ -245,6 +245,11 @@ With this update, you can provision IPv6 address spoke clusters from dual-stack
 
 With this release, you can pre-cache your application workload images before upgrading your applications on {sno} clusters with {cgu-operator-full}. For more information, see xref:../scalability_and_performance/ztp_far_edge/ztp-talm-updating-managed-policies.adoc#talm-prechache-user-specified-images-concept_ztp-talm[Pre-caching user-specified images with TALM on single-node OpenShift clusters].
 
+[id="ocp-4-14-ztp-siteconfig-disk-cleaning"]
+==== Disk cleaning option through SiteConfig and GitOps ZTP
+
+With this release, you can remove the partitioning table before installation by using the `automatedCleaningMode` field in the `SiteConfig` CR. For more information, see xref:../scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-sites.adoc#ztp-deploying-a-site_ztp-deploying-far-edge-sites[Deploying a managed cluster with SiteConfig and GitOps ZTP]
```
Comment (abhatt-rh) [inline release_notes/ocp-4-14-release-notes.adoc:251]:
```suggestion
With this release, you can remove the partitioning table before installation by using the `automatedCleaningMode` field in the `SiteConfig` CR. For more information, see xref:../scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-sites.adoc#ztp-deploying-a-site_ztp-deploying-far-edge-sites[Deploying a managed cluster with SiteConfig and GitOps ZTP].
```

**Review by abhatt-rh — COMMENTED** [review]
@amolnar-rh a minor nit to address before I can merge this one. Thanks!

**Pull request merged**